engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/DeploymentCreateProcessor.java

```diff
@@ -193,6 +193,21 @@ private void createDmnResources(
                 stateWriter.appendFollowUpEvent(command.getKey(), DecisionIntent.CREATED, record));
   }
 
+  /**
+   * Create a copy of the provided deployment record and write the Deployment:CREATED event with
+   * that copy
+   *
+   * @param command the already distributed command to process
+   * @param deploymentEvent the record to clone and modify
```

Comment:
>and modify

🔧 This reads as if the parameter may be modified as well, but only the copy is modified